justfile

```diff
@@ -234,6 +234,8 @@ generate-chinook-configuration: build start-dependencies
     echo "$(tput bold)$(tput setaf 3)WARNING:$(tput sgr0) Not updating the Aurora configuration because the connection string is unset."; \
   fi
 
+  ./scripts/archive-old-ndc-metadata.sh '{{POSTGRES_V3_CHINOOK_NDC_METADATA}}'
```

Comment:
I don't think there's much point to keeping these older versions around in this breaking-changes-epoch. (Even this PR makes changes to an archived version, which is non-sensical)

Reply:
Sure, I'll just remove it.